### ISSUE TITLE
 The loss is NaN because the validation set was specified as LMO instead of YCBV during training

### ISSUE THREAD

**Issue by CQUChenC** [nv-nguyen/gigapose#29]
Thank you for your excellent work!
During training, the validation set was set to LMO instead of YCBV as specified in the program, and the initial weights for the IST network were set to $ROOT_DIR/pretrained/loftr_indoor_ot.ckpt. During validation, an error occurred: the loss was NaN. The specific line is in loss.py at line 67: assert not torch.isnan(loss).

**Comment by GZWSAMA** [comment]
Hello，have you solved this problem?I have encountered the same problem with you.Thanks a lot
